Source/Blazorise/Modules/BaseJSModule.cs

```diff
@@ -192,5 +204,10 @@ private Task<IJSObjectReference> GetModule()
     /// </summary>
     protected IVersionProvider VersionProvider => versionProvider;
 
+    /// <summary>
+    /// Runs once after the module has been loaded.
+    /// </summary>
+    protected Func<IJSObjectReference, ValueTask> ModuleLoaded { get; set; }
```

Comment:
How about we convert this to virtual method `protected virtual ValueTask OnModuleLoaded`?

Reply:
fine by me